### PR TITLE
Adds starter files for resources jobs

### PR DESCRIPTION
**Pull request by ChatanyaSarin**

### CODE REVIEW

**Comment by trevorshannon** [comment]
Thanks! I'll try to take a look tomorrow.

File: resources-jobs/urlwatch.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    method: sendmail
+    sendmail:
+      path: /sbin/sendmail
+    subject: '{count} resource database changes detected ({datetime})'
+    details_url: https://storage.googleapis.com/ueo-changes/eligibility/report-{datetime}.html
+  gcs:
+    enabled: true
+    filename: report-{datetime}.html
+    bucket: ueo-changes
+    local_dir: ~/ueo-watch/eligibility/staging
```
Comment (trevorshannon) [inline resources-jobs/urlwatch.yaml:62]: this `eligibility` should be changed to `resources`, as well as below

File: resources-jobs/urlwatch.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    reported_field: customfield_10035
+    reviewer_field: customfield_10036
+    assignees:
+      - 712020:8ad994d4-719d-45fb-b332-ac14adae6a33  # Trevor
+      - 712020:bd72b02c-4651-44ef-bd26-418c1738cd14  # Chatanya Sarin
+    reviewers:
+      - 712020:8ad994d4-719d-45fb-b332-ac14adae6a33  # Trevor
+      - 712020:bd72b02c-4651-44ef-bd26-418c1738cd14  # Chatanya Sarin
+    error_assignee: 712020:8ad994d4-719d-45fb-b332-ac14adae6a33  # Trevor
+    details_url: https://storage.googleapis.com/ueo-changes/housing/report-{datetime}.html
```
Comment (trevorshannon) [inline resources-jobs/urlwatch.yaml:79]: Please replace `housing` with `resources`.

Sorry it has to be done in so many places...I think sometimes YAML supports variables

**Comment by ChatanyaSarin** [comment]
Hey Trevor, I updated the urlwatch.yaml file. It should be part of the original pull request now. Let me know if there are any other edits you would like for me to make.

**Review by trevorshannon — CHANGES_REQUESTED** [review]
Great, thanks!  It looks like the urls.yaml file did not get moved over properly.  If you just commit to your `main` branch with the urls.yaml move, we should be all set.

File: resources-jobs/url.yaml
Comment (trevorshannon) [inline resources-jobs/url.yaml:1]: Oh, and this should be urls.yaml (with an `s`)

**Review by trevorshannon — CHANGES_REQUESTED** [review]

**Review by trevorshannon — APPROVED** [review]
Thank you!

**Pull request merged**